### DISCUSSION TITLE
Froide fork for Switzerland

### DISCUSSION THREAD

**Discussion by loleg** [okfde/froide#496] [General]
Liebe Mitstreiter:innen, we [started a fork](https://github.com/OpendataCH/froide) last weekend, and I just saw that you have this area that we might use for a discussion about our experience. For my part, I wish to thank you - humbly and deeply - for all the efforts to make the platform work, and make it reusable. Froide and WhatDoTheyKnow are among my strongest motivators in the open data space - in these difficult times, more important than ever as beacons of hope and due diligence.

In the hackathon's [project page](https://challenges.openlegallab.ch/project/2) you can find links to a small splashy website that we built, and a notebook full of observations that could be useful feedback - or at least help us prioritise further investment in this area. These were based on what materials we could gather online and exploring your existing public platforms. Now that our own instance is up and running, things will get a little more interesting :)

**Comment by stefanw** [comment]
Hi and thanks for reaching out. We as OKF De hope to support you as much as we can.

Generally, I'd recommend not to fork `froide` – or at least not built customisations into froide – but use a separate Django project that uses the `froide` Django apps. Examples are the [German site repo `fragdenstaat_de`](https://github.com/okfde/fragdenstaat_de/) (that also includes lots of other stuff like our CMS, donation and newsletter system which you can ignore) and the [Austrian site repo `fragdenstaat_at`](https://github.com/fin/fragdenstaat_at).

The idea is to keep `froide` more general, configurable and fully translatable while the actual deployments are done with separate projects where we add other deployment-specific features, adjust settings, possibly add different translations and [override templates](https://docs.djangoproject.com/en/4.0/howto/overriding-templates/). This separation may be a bit more hassle up front, but will ensure that we that we can all benefit from one core code base.

We are also happy to accept PRs on `froide` to make it more customisable, e.g. add Django template `{% block %}`s for easier template overriding or add plugin hooks were needed.

If you are facing some architectural or specific challenges, please let me know. If you think a call would help, I'm up for it, too.

**Comment by loleg** [comment]
Sure - and that would be quite a good statement to put on the Readme & Getting Started guide. As we are still mostly just at the point where we are trying to figure out the details of the app, using the basic dev environment is fine. Your point is well taken, and once we have a roadmap in place, we will set up Django as CMS and your app inside it. The templating tips are much appreciated, though a headless app connecting to the API may be preferrable, depending on the way the frontend goes.

Something I couldn't figure out yet is if you have a Transifex site for the translations (this one looks [out of date](https://www.transifex.com/okfn/froide-2/)), or are using some other tool?

I found the Ansible setup, which is great, but is there by chance also a helm chart for kube deployment out there?
